CONTRIBUTING.md

```diff
@@ -63,15 +63,17 @@ Head to the Pages list table and drag and drop pages to change the order. Refres
 
 ### Release instructions
 
-1. Version bump: Bump the version number in `simple-page-ordering.php`.
-2. Changelog: Add/update the changelog in both `readme.txt` and `README.md`
-3. Readme updates: Make any other readme changes as necessary. `README.md` is geared toward GitHub and `readme.txt` contains WordPress.org-specific content. The two are slightly different.
-4. Merge: Make a non-fast-forward merge from `develop` to `master`.
-5. SVN update: Copy files over to the `trunk` folder of an SVN checkout of the plugin. If the plugin banner, icon, or screenshots have changed, copy those to the top-level `assets` folder. Commit those changes.
-6. SVN tag: Make a folder inside `tags` with the current version number, copy the contents of `trunk` into it, and commit with the message `Tagging X.Y.Z`. There is also an SVN command for tagging; however, note that it runs on the remote and requires care because the entire WordPress.org plugins repo is actually single SVN repo.
-7. Check WordPress.org: Ensure that the changes are live on https://wordpress.org/plugins/simple-page-ordering/. This may take a few minutes.
-8. Git tag: Tag the release in Git and push the tag to GitHub. It should now appear under [releases](https://github.com/10up/simple-page-ordering/releases) there as well.
-
-<p align="center">
-<a href="http://10up.com/contact/"><img src="https://10updotcom-wpengine.s3.amazonaws.com/uploads/2016/10/10up-Github-Banner.png" width="850"></a>
-</p>
+1. Branch: Starting from `develop`, cut a release branch named `release/X.Y.Z` for your changes.
+2. Version bump: Bump the version number in `package.json`, `readme.txt`, and `simple-page-ordering.php` if it does not already reflect the version being released.
+3. Changelog: Add/update the changelog in `readme.txt` and `CHANGELOG.md`.
+4. Props: update `CREDITS.md` file with any new contributors, confirm maintainers are accurate.
+5. Translations: Update the .pot file by running `npm run makepot`.
```

Comment:
Currently, we haven't defined npm scripts yet.

Reply:
@jeffpaul can you remove the pot file generation step or revert the last commit due to https://github.com/10up/simple-page-ordering/pull/45#discussion_r389207162

Reply:
_done_